migrations/migrations/005-templates-bearer-token.js

```diff
@@ -0,0 +1,5 @@
+const alterTemplatesTableToAddBearerToken = `
```

Comment:
This file seems unrelated to this PR.

Reply:
did not update the branch before creating it from main. Awesome catch !